frontend/tests/e2e/search/search-results.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { expect, test } from "@playwright/test";
+
+import {
+  fillSearchInputAndSubmit,
+  waitForSearchResultsInitialLoad,
+} from "./searchSpecUtil";
+
+test.beforeEach(async ({ page }) => {
+  const searchTerm = "grants";
+  await page.goto("/search");
+  await waitForSearchResultsInitialLoad(page);
+
+  // this is dumb but webkit has an issue with trying to fill in the input too quickly
+  // if the expect in here fails, we give it another shot after 5 seconds
+  // this way we avoid an arbitrary timeout, and do not slow down the other tests
+  try {
+    await fillSearchInputAndSubmit(searchTerm, page);
+  } catch (e) {
+    await fillSearchInputAndSubmit(searchTerm, page);
+  }
+
+  await page.waitForURL("/search?query=" + searchTerm);
+});
+
+test.afterEach(async ({ context }) => {
+  await context.close();
+});
+
+test.describe("Search page results tests", () => {
+  test("should return at least 1 result when searching with valid term", async ({
+    page,
+  }) => {
+    // eslint-disable-next-line testing-library/prefer-screen-queries
+    const resultsHeading = page.getByRole("heading", {
```

Comment:
this is a weird one - the eslint rule is coming from testing-library, but we're using playwright here, and the page object seems to just have methods like "getByRole" that are technically unrelated to testing-library but have the same names and functionality. In any case, I think we can probably work around this by using `page.locator` instead if we want to avoid the linting error